### ISSUE TITLE
Netflix issue 2024 g14

### ISSUE THREAD

**Issue by Need4Sun** [seerge/g-helper#2397]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

https://www.reddit.com/r/ZephyrusG14/comments/1b3w6bm/netflix_issue_2024_g14/

Someone posted this already but i was having the same problem, made an github account just to summit this.
Was having the same issue with netflix being out of sync. Went to the myasus software after restarting asus services in g helper and there was two updates that i think are responsable for the fix.

### How to reproduce the bug?

"When I play Netflix through the igpu(eco mode on g helper) the content plays in super speed and eventually runs into buffering and freezes. It then stutters like crazy."

I think the update that fixes the problem is the one called "HDR workaround". It doesn't appear on G helper so people might miss it so that is why i created the issue.

After installing both updates the problem was fixed. 

### Logs

2 item(s) updated successfully, 0 item(s) failed to update

Critical Updates (2)

[Ext] {52BDABF1-9B5F-4F72-AA44-A4EC37BB ...

Update successfully

Cirrus SmartAMP Extension 23.26.47.790

HDR workaround

Update successfully

ASUS Monitor Control

Last update time: 2024/04/06

V23.26.47.790

Driver

V1.0.13

AP

### Device and Model

g14 2024 GA403UV

### Armoury Crate

Uninstalled

### Asus Services

7

### Version

164

### OS

w11

**Comment by seerge** [comment]
@Need4Sun hello

Thanks for information

``Updates`` section is just a mirror of asus own website / api https://rog.asus.com/nl/laptops/rog-zephyrus/rog-zephyrus-g14-2024/helpdesk_download/ . I don't see any update called "HDR workaround" here :) so if you have a direct link you can post it

As for the rest, it's not a bug in G-Helper so I'm closing this as completed then as there is nothing I can "fix" here.

Thanks.

**Comment by Need4Sun** [comment]
The problem came back again so my guess was wrong. After checking everything, I discovered that Armory Crate was still installed on my ASUS G14 laptop. Although I had initially uninstalled it using the official software when I first bought the laptop, it seems that when I updated the PC (either via Windows or MyASUS), some parts of Armory Crate were reinstalled. I noticed a process called 'armory crate key control.exe' running, which surprised me. After running the official uninstaller again and rebooting, the problem was resolved, and Netflix ran perfectly.
